src/test/java/com/axis/eiffel/gerrit/lib/formatter/EiffelEventServiceTest.java

```diff
@@ -108,31 +111,31 @@ void tearDown() {
 
     private JsonObject getEiffelCreatedEvent() {
         return new JsonParser()
-                .parse("{\"meta\":{\"id\":\"14418a3d-5cf2-4d27-8b54-8d307b790164\",\"type\":\"EiffelSourceChangeCreatedEvent\",\"version\":\"4.0.0\",\"time\":1564659341925,\"tags\":[],\"source\":{\"domainId\":\"\",\"host\":\"\\\"https://link.se\\\"\",\"name\":\"\\\"Test Testsson\\\"\",\"uri\":\"\"}},\"data\":{\"gitIdentifier\":{\"commitId\":\"\\\"I4a75a2005033860f084463c15e7ce40312e53c40\\\"\",\"branch\":\"\\\"master\\\"\",\"repoName\":\"\\\"test\\\"\",\"repoUri\":\"\\\"https://link.se\\\"\"},\"customData\":[]},\"links\":[]}")
+                .parse(testDataHelper.getResourceAsString(EIFFEL_CREATED_EVENT))
                 .getAsJsonObject();
     }
```

Comment:
Now that we've extracted the test data to external files, how about not having a handful of nearly identical methods but rather one method that returns a JsonObject given a com.axis.eiffel.gerrit.lib.formatter.TestDataHelper.ResourceType as input? Actually, the JSON loading could be in the ResourceType class itself, no? Allowing e.g.

    JsonObject o = EIFFEL_SUBMITTED_EVENT.load()

?

Reply:
That's actually a great idea, thank you! Done it